github-connector/cmd/github-connector/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/githubwrappers"
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/handlers"
+)
+
+func main() {
+
+	log.Println("server started")
+	webhook := handlers.NewWebHookHandler(
+		githubwrappers.ReceivingEventsWrapper{},
+	)
+
+	http.HandleFunc("/webhook", webhook.HandleWebhook)
+	log.Fatal(http.ListenAndServe(os.Getenv("PORT"), nil))
```

Comment:
please use `log.Info` here:
```suggestion
	log.Info(http.ListenAndServe(os.Getenv("PORT"), nil))
```

Reply:
(this is the place where the colon needs to be added to the port number by the way)